components/camel-pulsar/src/main/java/org/apache/camel/component/pulsar/utils/message/PulsarMessageHeaders.java

```diff
@@ -31,4 +31,5 @@
     String KEY_OUT = "CamelPulsarProducerMessageKey";
     String PROPERTIES_OUT = "CamelPulsarProducerMessageProperties";
     String EVENT_TIME_OUT = "CamelPulsarProducerMessageEventTime";
+    String PULSAR_REDELIVERY_COUNT = "pulsar_redelivery_count";
```

Comment:
Please use the same Camel naming approach for the headers. Like for example CamelPulsarRedeliveryCount